daniellamarr/fastfoodfast-reactjs
# #164399457 User can add to cart

#### What does this PR do?
- This PR allows a user to add food items to his cart
#### Description of Task to be completed?
- A user should be able to add a food item to his cart
- A user can see the food items in the cart
#### How should this be manually tested?
- Install dependencies `npm install`
- Start the development server `npm start`
- Log into the app and click on the add to cart button to add food items to cart
- Click on the cart icon on the header to see products added to the cart
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164399457

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2019-03-06 at 12 26 35 pm" src="https://user-images.githubusercontent.com/42177767/53878048-2751c900-400b-11e9-836f-d2d6a8b26c35.png">

#### Questions:
- N/A
